dashboard/modules/serve/sdk.py

```diff
@@ -77,14 +78,21 @@ def deploy_application(self, config: Dict) -> None:
         if response.status_code != 200:
             self._raise_error(response)
 
-    def get_info(self) -> Union[Dict, None]:
+    def get_info(self) -> Dict:
         response = self._do_request("GET", INFO_PATH)
         if response.status_code == 200:
             return response.json()
         else:
             self._raise_error(response)
 
-    def get_status(self) -> Union[Dict, None]:
+    def get_serve_details(self) -> ServeInstanceDetails:
+        response = self._do_request("GET", "/api/serve/applications/")
+        if response.status_code == 200:
+            return ServeInstanceDetails(**response.json())
```

Comment:
Could you make this function return a `Dict` and make the caller convert it to a `ServeInstanceDetails`, so it matches the rest of the SDK?

Reply:
Resolved offline– we'll keep it as `ServeInstanceDetails`.